### PR TITLE
sci-biology/psychopy version bump

### PR DESCRIPTION
**Pull request by TheChymera**
1.83.03

### CODE REVIEW

File: sci-biology/psychopy/psychopy-1.83.03.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+DESCRIPTION="Python experimental psychology toolkit"
+HOMEPAGE="http://www.psychopy.org/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/psychopy/psychopy.git"
+EGIT_COMMIT="1.83.03"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (jlec) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: No KEYWORDS in live ebuilds.

File: sci-biology/psychopy/psychopy-1.83.03.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
+RDEPEND="${DEPEND}
```
Comment (jlec) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: Is setuptools needed during runtime?

File: sci-biology/psychopy/psychopy-1.83.03.ebuild (hunk 1 above)
Comment (TheChymera) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: This is not live, it's already versioned.

Comment (jlec) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: live means, you cannot install it without internet connection. In fact, you also need to be able to reach Github. Either do a proper snapshot, pull a snapshot from a tag or drop KEYWORDS.

Comment (TheChymera) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: How do I pull a snapshot from a tag? Apparently there is an archive with the release at this address: https://github.com/psychopy/psychopy/releases/download/1.83.03/PsychoPy-1.83.03.zip Can I link to that with egit? or should I use SRC_URI?

Comment (jlec) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: This is an zip file, so it should be in SRC_URI.

Comment (TheChymera) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: @jlec will that count as non-live?

Comment (jlec) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: sure.

Comment (TheChymera) [inline sci-biology/psychopy/psychopy-1.83.03.ebuild]: @jlec done

**Pull request merged**